.gitignore

```diff
@@ -1,5 +1,7 @@
 # dependencies
 **/node_modules
+package-lock.json
```

Comment:
I was just talking with @AkiraLaine about the `yarn.lock` file actually, didn't realize it was the same thing and NPM 5s new `package-lock.json`(which I guess is the same as the old shrink-wrap file which I didn't even know about). 

Seeing that this project was not created with NPM 5, would there be any conflict to you think for actually committing this instead? I guess it may help newer contributors who are using NPM 5 to definitely get all the right versions of the packages.